docs/fragments/pr-sanitizer.adoc

```diff
@@ -0,0 +1,37 @@
+== PR Sanitizer Plugin
+
+This plugin enforces writing effective pull requests that confirm with proper naming conventions and description to clearly convey its purpose and provide better collaboration, tracebility and maintainabilty by giving a clear and concise context to the reviewer of what’s being solved at a high level, taking off an extra step for the reviewer.
+
+=== Title Verification [[title-verification]]
+
+This ensures that the PR title comforms with the semantic commit message style. Conformance with the semantic commit message style not only makes your changelog and git history clean and easy to navigate but also encourages making atomic pull requests that address only a single concern, better reflect the change within and are more clearly understood.
+
+The format below shows a rigid semantic commit message style.
+
+[source, shell, indent=0]
+----
+<type>(<scope>): <subject>
+----
+
+* where `<scope>` is optional.
+* and each of `<type>` categorizes the PR according to the change introduced (such as `feat` for introducing a new feature and `fix` for patching a bug in codebase)
+ 
+If the PR title is prefixed with any of the default types (`feat`, `fix`, `docs`, `style`, `refactor`, `test`, `chore`) or any of the user configured types as described below (non-case sensitive) then the plugin marks its check as **Success**, or as **Failure** otherwise.
```

Comment:
Here, I would probably also attach a mention that we were inspired by the https://seesparkbox.com/foundry/semantic_commit_messages page - there is also the explanation of all types